web/proxy.js

```diff
@@ -31,6 +31,7 @@ const server = http.createServer((request, response) => {
         headers: {
             ...request.headers,
             host,
+            'user-agent': request.headers['user-agent'].concat(' Development-NewDot/1.0'),
```

Comment:
is this space prefix in  `' Development-NewDot/1.0'` on purpose?

Reply:
Yes, it's just the separator because it's appending to the existing User Agent string

Reply:
Thanks for the explanation!